downstream/modules/platform/ref-system-requirements.adoc

```diff
@@ -37,7 +37,7 @@ The following are necessary for you to work with project updates and collections
 [NOTE]
 ====
 The requirements for systems managed by {PlatformNameShort} are the same as for Ansible.
-See link:https://docs.ansible.com/ansible/latest/user_guide/intro_getting_started.html[Getting Started] in the Ansible _User Guide_.
+See link:https://docs.ansible.com/ansible/latest/getting_started/index.html[Getting started with Ansible] in the Ansible _User Guide_.
```

Comment:
If you look at the "rich text" view, you can see that this note is identical to the bullet point beneath it. Should we remove one or the other? I would probably go with removing the bullet point.

Reply:
Deleted bullet point